src/index.css

```diff
@@ -238,6 +238,73 @@ nav svg {
   font-size: 1.25rem;
 }
 
+/*
+===============
+Book Form
+===============
+*/
+hr {
+  border: solid 0.5px #e8e8e8;
+}
+
+.foot-title {
+  font-family: var(--font-mont);
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
```

Comment:
- [ ] I suggest that you adjust the style (font-color, font-weight and font-size) of nav items (`Books` and `Categories`) to match the given design :blush: 

- [ ] Please make sure that you remove the text `YOUR BAG` from your design since you are asked to follow the Zeplin design which has no such text above the container of the books as you can see  below :point_down: 

### Your design
![image](https://github.com/addisalemseifu/bookstore-app-with-react/assets/73854999/af4f1ec1-0927-494a-9e21-7f06407040e1)


### Expected design
![image](https://github.com/addisalemseifu/bookstore-app-with-react/assets/73854999/7b0c3b99-1225-4d94-912d-1ebb9bdf12aa)